extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -0,0 +1,603 @@
+			}).component();
+		this._networkShareContainerStorageAccountDropdown = view.modelBuilder.dropDown()
+			.withProps({
+				required: true
+			}).component();
+		this._networkShareContainerStorageAccountDropdown.onValueChanged((value) => {
+			this._networkShare.storageAccountId = (this._networkShareContainerStorageAccountDropdown.value as azdata.CategoryValue).name;
+		});
+
+		const flexContainer = view.modelBuilder.flexContainer().withItems(
+			[
+				networkShareHelpText,
+				networkShareLocationLabel,
+				this._networkShareLocationText,
+				windowsUserAccountLabel,
+				this._windowsUserAccountText,
+				passwordLabel,
+				this._passwordText,
+				azureAccountHelpText,
+				subscriptionLabel,
+				this._networkShareContainerSubscriptionDropdown,
+				storageAccountLabel,
+				this._networkShareContainerStorageAccountDropdown
+			]
+		).withLayout({
+			flexFlow: 'column'
+		}).component();
+
+		return flexContainer;
+	}
+
+	private emailNotificationContainer(view: azdata.ModelView): azdata.FormComponent {
+		const emailCheckbox = view.modelBuilder.checkBox().withProps({
+			label: constants.DATABASE_BACKUP_EMAIL_NOTIFICATION_CHECKBOX_LABEL
+		}).component();
+
+		emailCheckbox.onChanged((value) => this.migrationStateModel.databaseBackup.emailNotification = value);
+
+		return {
+			title: constants.DATABASE_BACKUP_EMAIL_NOTIFICATION_LABEL,
+			component: emailCheckbox
+		};
+	}
+
+	private migrationCutoverContainer(view: azdata.ModelView): azdata.FormComponent {
+		const description = view.modelBuilder.text().withProps({
+			value: constants.DATABASE_BACKUP_MIGRATION_CUTOVER_DESCRIPTION
+		}).component();
+
+		const buttonGroup = 'cutoverContainer';
+
+		const automaticButton = view.modelBuilder.radioButton().withProps({
+			label: constants.DATABASE_BACKUP_MIGRATION_CUTOVER_AUTOMATIC_LABEL,
+			name: buttonGroup,
+			checked: true
+		}).component();
+
+		this.migrationStateModel.databaseBackup.migrationCutover = MigrationCutover.AUTOMATIC;
+
+		automaticButton.onDidClick((e) => this.migrationStateModel.databaseBackup.migrationCutover = MigrationCutover.AUTOMATIC);
+
+		const manualButton = view.modelBuilder.radioButton().withProps({
+			label: constants.DATABASE_BACKUP_MIGRATION_CUTOVER_MANUAL_LABEL,
+			name: buttonGroup
+		}).component();
+
+		manualButton.onDidClick((e) => this.migrationStateModel.databaseBackup.migrationCutover = MigrationCutover.MANUAL);
+
+		const flexContainer = view.modelBuilder.flexContainer().withItems(
+			[
+				description,
+				automaticButton,
+				manualButton
+			]
+		).withLayout({
+			flexFlow: 'column'
+		}).component();
+
+		return {
+			title: constants.DATABASE_BACKUP_MIGRATION_CUTOVER_LABEL,
+			component: flexContainer
+		};
+	}
+
+	public async onPageEnter(): Promise<void> {
+		await this.getSubscriptionValues();
+		this.wizard.registerNavigationValidator((pageChangeInfo: azdata.window.WizardPageChangeInfo) => {
+			return true;
+		});
+		return Promise.resolve();
```

Comment:
remove

Reply:
(and in async functions you should not be returning a Promise.resolve like this - it'll wrap return values for you already)